components/scheduler/events.md

```diff
@@ -598,6 +601,225 @@ To implement appointment editing, the scheduler exposes the `OnCreate`, `OnDelet
 }
 ````
 
+## OnItemClick
+
+The `OnItemClick` event fires when the user clicks on an appointment in the Scheduler. 
+It provides a `SchedulerItemClickEventArgs` object to the event handler and you can get the `Item` and cast it to your own model. If you set the `ShouldRender` field to `true`, the component will re-render. This can be useful if you need to change its parameters or state during the event execution and especially if you need to execute `async` logic in the event handler.
```

Comment:
https://stackoverflow.com/questions/295104/what-is-the-difference-between-a-field-and-a-property

```suggestion
It provides a `SchedulerItemClickEventArgs` object to the event handler and you can get the `Item` property and cast it to your own model. If you set the `ShouldRender` property to `true`, the component will re-render. This can be useful if you need to change the Scheduler parameters or state during the event execution and especially if you need to execute `async` logic in the event handler.
```